src/clj/harja/palvelin/raportointi/raportit/laskutusyhteenveto_tuotekohtainen.clj

```diff
@@ -80,18 +80,22 @@
                                (rivi-taulukolle data kyseessa-kk-vali? "Sanktiot" :sakot_laskutettu :sakot_laskutetaan false)
 
                                ;; Hoitovuoden päättäminen, näytetään vain jos arvot olemassa 
-                               (when (raha-arvo-olemassa? (data :hj_hoitovuoden_paattaminen_tavoitepalkkio_laskutettu))
-                                 (rivi-taulukolle data kyseessa-kk-vali? "Hoitovuoden päättäminen / Tavoitepalkkio"
-                                                  :hj_hoitovuoden_paattaminen_tavoitepalkkio_laskutettu :hj_hoitovuoden_paattaminen_tavoitepalkkio_laskutetaan false))
-
-                               (when (raha-arvo-olemassa? (data :hj_hoitovuoden_paattaminen_tavoitehinnan_ylitys_laskutettu))
-                                 (rivi-taulukolle data kyseessa-kk-vali? "Hoitovuoden päättäminen / Urakoitsija maksaa tavoitehinnan ylityksestä"
-                                                  :hj_hoitovuoden_paattaminen_tavoitehinnan_ylitys_laskutettu :hj_hoitovuoden_paattaminen_tavoitehinnan_ylitys_laskutetaan false))
-
-                               (when (raha-arvo-olemassa? (data :hj_hoitovuoden_paattaminen_kattohinnan_ylitys_laskutettu))
-                                 (rivi-taulukolle data kyseessa-kk-vali? "Hoitovuoden päättäminen / Urakoitsija maksaa kattohinnan ylityksestä"
-                                                  :hj_hoitovuoden_paattaminen_kattohinnan_ylitys_laskutettu :hj_hoitovuoden_paattaminen_kattohinnan_ylitys_laskutetaan false))
-
+                               (try
+                                 (when (raha-arvo-olemassa? (data :hj_hoitovuoden_paattaminen_tavoitepalkkio_laskutettu))
```

Comment:
Pikkuasia: `data` on ilmeisesti map. Tyypillisempää olisi kutsua `(:avain map)` vs. `(map :avain)`. Funktionaalisesti sama asia, mutta näyttää omaan silmään hieman hassulta näin päin tehtynä

Reply:
https://github.com/finnishtransportagency/harja/pull/2824/commits/4430c3ac4f260caa3c7d09940b217c5ba84011d9